spot_driver/src/api/default_time_sync_api.cpp

```diff
@@ -6,14 +6,15 @@
 
 namespace spot_ros2 {
 
-DefaultTimeSyncApi::DefaultTimeSyncApi(std::shared_ptr<::bosdyn::client::TimeSyncThread> time_sync_thread)
-    : time_sync_thread_{time_sync_thread} {}
+DefaultTimeSyncApi::DefaultTimeSyncApi(std::shared_ptr<::bosdyn::client::TimeSyncThread> time_sync_thread,
+                                       int8_t timesync_timeout)
```

Comment:
```suggestion
                                       std::chrono::milliseconds timesync_timeout)
```
Doesn't have to be milliseconds, but chrono instead of int8_t seems more appropriate here